contributors/devel/architectural-roadmap.md

```diff
@@ -829,9 +829,9 @@ therefore wouldn’t be considered to be part of Kubernetes.
       a spectrum of possible solutions, each with different
       tradeoffs. That said, common building blocks (e.g., [secure
       Kubelet
-      registration](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/kubelet-tls-bootstrap.md))
+      registration](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/kubelet-tls-bootstrap.md))
       and approaches (in particular,
-      [self-hosting](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/self-hosted-kubernetes.md#what-is-self-hosted))
+      [self-hosting](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/cluster-lifecycle/self-hosted-kubernetes.md#what-is-self-hosted))
```

Comment:
same repo. This can be:
`[self-hosting](/contributors/design-proposals/cluster-lifecycle/self-hosted-kubernetes.md#what-is-self-hosted))`